bundles/org.pitest.pitclipse.launch.ui/META-INF/MANIFEST.MF

```diff
@@ -16,6 +16,7 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="[3.11.1,4.0.0)",
  org.pitest.pitclipse.launch,
  org.eclipse.jface;bundle-version="[3.11.1,4.0.0)",
  org.eclipse.ui.workbench;bundle-version="[3.107.1,4.0.0)",
- org.eclipse.jdt.debug.ui;bundle-version="[3.7.101,4.0.0)"
+ org.eclipse.jdt.debug.ui;bundle-version="[3.7.101,4.0.0)",
+ org.pitest;bundle-version="1.6.7"
```

Comment:
Maybe reason tests on Mac are failing?!

Reply:
macOS builds started to fail yesterday also on my branches with no expected reasons... I also tried to re-run old successful builds (e.g., on master) and they always fails with a NPE as soon as UI tests start... I guess it's a problem with the macOS virtual environment. It might also be due to the fact we are using a very old Eclipse (Photon) in the target platform... I don't know whether I'm able to look into this in the next few days.

Reply:
Thanks for the heads-up.